tests/functional/test_timeformat.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import os
+import mock
+import json
+import datetime
+from dateutil.tz import tzlocal
+from awscli.testutils import unittest, FileCreator, capture_output
+from awscli.clidriver import create_clidriver
+
+
+class TestCLITimestampParser(unittest.TestCase):
+    def setUp(self):
+        self.files = FileCreator()
+        self.environ = {
```

Comment:
Originally yes, but there has been some refactoring so I'll add something to testutils for this mocking `botocore.endpoint.Session.send` in general.